stackslib/src/chainstate/stacks/boot/pox_4_tests.rs

```diff
@@ -7745,23 +8113,24 @@ fn test_scenario_three() {
         .unwrap();
     assert_eq!(david_aggregate_commit_indexed_err, Value::Int(35));
 
+    let david_index = if use_nakamoto { 3 } else { 2 };
```

Comment:
Why is this?

Reply:
Similar to the other issue - when in Nakamoto, there is a default stacker/signer.